### ISSUE TITLE
[feature request] remove unused vendored package in pouch

### ISSUE THREAD

**Issue by allencloud** [alibaba/pouch#2399]
## Why you need it?

When I learn from this project to help [Dragonfly](https://github.com/alibaba/Dragonfly) to support swagger. I found that in there are so many vendored packages in Pouch. I have some doubt that whether all these package in vendor.json is all used in the project.

When I use `govendor list +unused`, it lists : 

```
pouch (master) $ govendor list +unused
 vu github.com/containerd/containerd/version  v1.0.3  v1.0.3
 vu github.com/go-ini/ini
 vu github.com/go-resty/resty
 vu github.com/magiconair/properties
 vu github.com/russross/blackfriday
 vu github.com/spf13/pflag
 vu github.com/vishvananda/netns
 vu golang.org/x/sys/unix
```

It seem some packages are unnecessary. Please help to remove that to keep vendor packages pure.

## How it could be?

no unused packages located in vendor folder.

## Other related information

none

**Comment by fengzixu** [comment]
I would like to handle it. @allencloud

**Comment by allencloud** [comment]
Thanks @fengzixu 
Please feel free to tell us if you need any help.

**Comment by fengzixu** [comment]
I try to clear the unused packages. 

#### Before modification

```
 vu github.com/containerd/containerd/version  v1.0.3  v1.0.3
 vu github.com/go-ini/ini
 vu github.com/go-resty/resty
 vu github.com/magiconair/properties
 vu github.com/russross/blackfriday
 vu github.com/spf13/pflag
 vu github.com/vishvananda/netns
 vu golang.org/x/sys/unix
```

#### After modification


##### Packages below were used in other vendors packages
```
vu github.com/russross/blackfriday
vu github.com/vishvananda/netns
```

##### Packages below were used in pouch packages
`vu github.com/spf13/pflag` ====> https://github.com/alibaba/pouch/blob/master/cli/common_flags.go#L6
`vu golang.org/x/sys/unix`  =====>  https://github.com/alibaba/pouch/blob/master/cri/v1alpha1/cri_utils.go#L29


I consider that those packages were identified by govendor mistakely.

**Comment by fuweid** [comment]
closed by  #2404
